packages/gatsby-remark-prismjs/README.md

```diff
@@ -27,6 +27,13 @@ plugins: [
             // This is an uncommon use-case though;
             // If you're unsure, it's best to use the default value.
             classPrefix: "language-",
+            // This is used to allow setting a language for inline code
```

Comment:
Could you add an example of how this works? I had to read the code before I realized that the `inlineCodeMarker` let you do something like `javascript>let p = "blue"`

Reply:
Sure. Gimme a second.

Reply:
Done!